### DISCUSSION TITLE
An sqlite only version ?

### DISCUSSION THREAD

**Discussion by fadi123go** [dbeaver/dbeaver#23186] [Ideas]
I know dbeaver is a universal database manager but could you create an sqlite only version, easy to use yet featureful enough for desktop use only . If it is not within the scope of this project would someone reading this post create a fork. Such version would compete with Ms Access and Base , and be more attractive to spreadsheet users who are less used to dealing with servers and more used to dealing with files. You could call this version dbeaver lite or dbeaver desktop.

**Comment by LonwoLonwo** [comment]
Hello @fadi123go 

We do not have plans to create products only for one database. Because DBeaver is multibase manager tool.

But we already have our Lite Edition: https://dbeaver.com/dbeaver-lite/

Or, maybe CloudBeaver Edition will be more handy for you: https://github.com/dbeaver/cloudbeaver/
